jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/controller/ResourceApiControllerTest.java

```diff
@@ -241,8 +244,27 @@ void create_idProvidedInPayload_illegalArgumentExceptionThrown() {
     @Test
     void create_payloadViolatesEntityConstraints_resourceConstraintViolationExceptionThrown() {
         var controller = getResourceApiController(DummyEntityD.class);
-        assertThatExceptionOfType(ResourceConstraintViolationException.class).isThrownBy(() -> controller.create(TENANT_ID, "{}"))
-                .withMessageContainingAll("description", "telephone", "has the following error(s):");
+        Throwable thrown = catchThrowable(() -> controller.create(TENANT_ID, "{}"));
+
+        assertThat(thrown).isInstanceOf(ResourceConstraintViolationException.class);
+        var errorResponse = (((ResourceConstraintViolationException) thrown).getErrorResponse());
```

Comment:
nit: think there's a extra pair of brackets here.

Reply:
Fixed